bazel/v8.patch

```diff
@@ -9,7 +9,7 @@ index 4e89f90..0df4f67 100644
 --- a/BUILD.bazel
 +++ b/BUILD.bazel
 @@ -4,7 +4,7 @@
- 
+
```

Comment:
Can we revert the white-space changes in this file?

Reply:
yes, done